channels/pe.m3u

```diff
@@ -11,11 +11,29 @@ http://cdnh4.iblups.com/hls/OVJNKV4pSr.m3u8
 https://cdnhd.iblups.com/hls/ptv2.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://i.imgur.com/B89KuHO.png" group-title="",Pax TV (480p)
 https://d2fxrfbiedz1tm.cloudfront.net/livepaxtv/smil:PC.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/uncanalinteligente/picture?width=320&height=320" group-title="",UCI
+http://cdn.win.pe:8080/UCI/tracks-v1a1/mono.m3u8
```

Comment:
Link doesn't work.
<img width="701" alt="Captura de pantalla 2021-02-20 a las 14 38 58" src="https://user-images.githubusercontent.com/7253922/108594194-812f3300-7389-11eb-8e0d-e9cc4b8f85ed.png">

Reply:
Updated and fixed: http://170.79.102.223/UCI/index.m3u8